install/kubernetes/tetragon/Chart.yaml

```
@@ -3,6 +3,7 @@ name: tetragon
 description: Helm chart for Tetragon
 type: application
 version: 1.4.0
+icon: https://cilium.io/static/tetragon-shield-9b9d3b1f08397c40c37a12f8e5b442b0.png
```

sure I don't mind! Why not using tetragon.io static directly :)?
```suggestion
icon: https://tetragon.io/images/tetragon-shield.png
```